xml/System.Collections.Generic/Dictionary`2.xml

```diff
@@ -331,8 +331,8 @@
         <Parameter Name="collection" Type="System.Collections.Generic.IEnumerable&lt;System.Collections.Generic.KeyValuePair&lt;TKey,TValue&gt;&gt;" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
       </Parameters>
       <Docs>
-        <param name="collection">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="collection">The <see cref="T:System.Collections.Generic.IEnumerable`1" />  whose elements are copied to the new <see cref="T:System.Collections.Generic.Dictionary`2" />.</param>
```

Comment:
In the MS Docs page for this API, the data type will already be showing up next to the parameter, so there's no need to indicate it again with a cref. The same goes for the type: you already know you're looking at an API of the Dictionary type. So instead, it's preferable to use natural language, like this:

```suggestion
        <param name="collection">The enumeration whose elements are copied to the new dictionary.</param>
```

It's not mandatory, but preferred. So feel free to apply (or not) a similar change for all the APIs in this PR.

Reply:
I considered doing this, but decided not to in order to be consistent with existing API docs for these types & not cause a huge diff here to unify them. I'll leave as is for now.